content/other/phase-angles/cos/features/dataset-attributes.md

```diff
@@ -0,0 +1,7 @@
+|Name|Type|Description|
+|-|-|-|
+|angles|dict|Angles that apply the polynomial transformation for a given routine. This is provided as a dictionary of lists with form `angles[routine][epsilon][k][angle_index]`.|
```

Comment:
Would you say it is necessary "angle_index" and "polynomial_index" bellow?

Reply:
Not necessary, but consistent with 1/x from before. We could change all of them :thinking: